packages/material-ui/src/SubmitButton/SubmitButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Box from "@material-ui/core/Box";
```

Comment:
@epicfaace The pattern for `@material-ui` is different, we don't import directly from `@material-ui/core` anymore.